src/sendgrid.lisp

```diff
@@ -84,10 +87,17 @@ The JSON looks like:
   `reply-to': a pair of email and name.
   `from': the sending email
   `from-name': the sending name that shows up in the inbox"
-  (assert (and to
-               from
-               subject
-               content-value))
+  (if attachments
+      (assert (and to
+		   from
+		   subject
+		   content-value
+		   file
+		   filename))
```

Comment:
I think we can do better, like 1 assert and then the other, but mostly return an explicit error to the user: if attachments is T and no file or filename is given, explain.

But that's out of stock of this PR, so no pb.

Reply:
Actually, I am handling the specific error handling of this entire project in the third update we discussed. I can, of course, add this specific change here.

What do you mean by "so a change is out of stock of this PR?"

Reply:
In the error handling PR upgrade, I planned on removing asserts entirely and moving to handling errors.

Reply:
great! 
> out of scope for this PR
I meant that this PR is about txt attachments, not better errors, so I'm alright to leave it as is now.

so, all good here.